spec/bolt/node/winrm_spec.rb

```diff
@@ -58,6 +46,18 @@ def stub_winrm_to_raise(klass, message)
       end
     end
 
+    it "raises Node::ConnectError if the node name can't be resolved" do
```

Comment:
If I run this test first, it fails because it takes > 1 second to fail (11 seconds), implying it hit the timeout. However, the assertions pass, implying that the correct exception is raised.

If I don't run it first, it works just fine. That's cool.

Reply:
I increased the default timeout to 20 seconds and this still fails after 11. That means it must not be the bolt timeout it's waiting for (a relief, since it's not raising a timeout error...), but some other DNS timeout. One mystery down, but that still leaves open the question of why reordering the test fixes it (as does including a `winrm.connect` at the beginning...).